redisvl/extensions/llmcache/semantic.py

```diff
@@ -320,15 +315,18 @@ def check(
         cache_search_results = self._index.query(query)
 
         for cache_search_result in cache_search_results:
-            key = cache_search_result["id"]
-            self._refresh_ttl(key)
+            redis_key = cache_search_result.pop("id")
```

Comment:
is this related of just a bonus part of pr?

Reply:
I found a small hidden bug that broke my tests after the query refactor work. The new query refactor work changes the way the return_fields are generated/stored on the class and that exposed an issue in how the cache checks are done